tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -579,9 +579,52 @@ public void Builds_AzureFunctionsProject()
             var appDir = volume.ContainerDir;
             var appOutputDir = "/tmp/AzureFunctionsHttpTriggerApp-output";
             var script = new ShellScriptBuilder()
+                .SetEnvironmentVariable("Oryx_App_type", "Functions")
```

Comment:
Can you also add unit tests for the new wasm project detection in `ProjectFileHelpersTest`?

Reply:
sure